NEWS

```diff
@@ -30,6 +42,10 @@
           o SipHash
           o ARIA (including TLS support)
       o Significant Side-Channel attack security improvements
+      o Add a new ClientHello callback. Provides a callback interface that gives
+        the application the ability to adjust the nascent SSL object at the
+        earliest stage of ClientHello processing, immediately after extensions
+        have been collected but before they have been processed.
```

Comment:
Gosh, this text even sounds like it was written by me ;)

One potentially noteworthy aspect (and a recurring feature request, IIUC) is that this allows the callback to affect TLS version negotiation, something that was previously impossible for application code to do.  So maybe, "In particular, this callback can adjust the supported TLS versions in response to the contents of the ClientHello".  But maybe that's too much detail; I think I'm too close to the work to be an objective arbiter.

Reply:
I'll add that to CHANGES and slim down the NEWS entry as per the comment by @levitte.